app/assets/stylesheets/application.scss

```diff
@@ -66,6 +66,14 @@ html {
 }
 
 .toast {
-  &.success { background-color: $success-color; }
-  &.error { background-color: $error-color; }
+  background-color: #F6F6F6;
```

Comment:
Color literals like `#F6F6F6` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F6F6F6` should be written as `#f6f6f6`